modules/virt-about-efi-mode-for-vms.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * virt-efi-mode-for-vms.adoc
+
+[id="virt-about-efi-mode-for-vms_{context}"]
+= About EFI mode for virtual machines
+
+`EFI` (Extensible Firmware Interface) is a firmware interface that initializes the hardware components and starts the operating system stored on a hard disk drive when a computer starts.
+
+It stores all the information about initialization and startup in a .efi file, which is stored on a special partition called EFI System Partition (ESP). The ESP also contains the boot loader programs for the operating system installed on the computer.
+
+The ESP allows `EFI` to directly boot the operating system and save the BIOS self-test process, allowing the computer to boot faster.
```

Comment:
no backticks around EFI

What does "save" mean here?

Reply:
If faster booting is the main reason for using EFI mode, I think you should mention that earlier in the module.